packages/cactus-cmd-api-server/README.md

```diff
@@ -8,10 +8,15 @@
 - [Deployment Scenarios](#deployment-scenarios)
```

Comment:
@jordigiam Same as for the Dockerfile above

Reply:
Done